src/palace/manager/feed/annotator/circulation.py

```diff
@@ -740,6 +740,18 @@ def __init__(
         self._top_level_title = top_level_title
         self.identifies_patrons = library_identifies_patrons
         self.facets = facets or None
+        self._is_novelist_configured: bool | None = None
+
+    @property
```

Comment:
Minor: could use `cached_property` if you don't need to control the caching here

Reply:
Good point! I'll do that.